src/pages/plan/pdf/PDFTemplate.tsx

```
@@ -1,7 +1,10 @@
 import { Document, Page, Text, View, StyleSheet, Image, Font } from '@react-pdf/renderer';
-import type { Schedule, Traveler } from './types/PDFDataType';
+import type { Schedule } from './types/PDFDataType';
 import { useEffect, useState } from 'react';
 import QRCode from 'qrcode';
+import type { TravelerType } from '@/api/types/traveler';
+import type { PlanDetailType } from '@/api/types/planDetail';
+import { schedules, travelers } from './data/example';
```

ESLint

[error] 'travelers' is defined but never used.

(@typescript-eslint/no-unused-vars)